retrievalmarket/storage_retrieval_integration_test.go

```diff
@@ -220,6 +222,8 @@ func newStorageHarness(ctx context.Context, t *testing.T) *storageHarness {
 	)
 	require.NoError(t, err)
 	dt2 := graphsyncimpl.NewGraphSyncDataTransfer(td.Host2, td.GraphSync2, td.DTStoredCounter2)
+	storedAsk, err := storedask.NewStoredAsk(td.Ds2, datastore.NewKey("latest-ask"), providerNode, providerAddr)
+	assert.NoError(t, err)
```

Comment:
It's a nitpick but I think this should be a require.NoError since the test doesn't make sense unless this succeeds.